packages/lit-html/src/test/directives/unsafe-svg_test.ts

```diff
@@ -33,6 +33,38 @@ suite('unsafeSVG', () => {
     assert.equal(lineElement.namespaceURI, 'http://www.w3.org/2000/svg');
   });
 
+  test('renders nothing', () => {
```

Comment:
Same suggestions as the `unsafeHTML` tests.

Reply:
Done